isbn10to13bot/isbn_10_to_13.py

```diff
@@ -0,0 +1,80 @@
+"""
+isbn 10 to isbn 13
+NOTE: This script ideally works on an Open Library Dump that only contains editions with an isbn_10 and no isbn_13
+"""
+import gzip
+import json
+
+import isbnlib
+import olclient
+
+
+class ConvertISBN10to13Job(olclient.AbstractBotJob):
+    def run(self) -> None:
+        """Looks for any ISBN 10s to convert to 13"""
+        self.write_changes_declaration()
+        header = {"type": 0, "key": 1, "revision": 2, "last_modified": 3, "JSON": 4}
+        comment = "convert ISBN 10 to 13 using isbnlib"
+        with gzip.open(self.args.file, "rb") as fin:
+            for row_num, row in enumerate(fin):
+                row = row.decode().split("\t")
+                _json = json.loads(row[header["JSON"]])
+                if _json["type"]["key"] != "/type/edition":
+                    continue
+
+                if "isbn_10" in _json:
+                    isbns_10 = _json.get("isbn_10", None)
+                else:
+                    # we only update editions with existing isbn 10s
+                    continue
+
+                if "isbn_13" in _json:
+                    # we only update editions with no existing isbn 13s (for now at least)
+                    continue
+
+                olid = _json["key"].split("/")[-1]
+                edition = self.ol.Edition.get(olid)
+                if edition.type["key"] != "/type/edition":
+                    continue
+
+                if hasattr(edition, "isbn_13"):
+                    # we only update editions with no existing isbn 13s (for now at least)
+                    continue
+
+                isbns_13 = []
```

Comment:
It is too easy to overlook the plural form when it is in the middle of the variable name.
```suggestion
                isbn_13s = []
```

Reply:
I avoided that becausi it might randomise the order of the existing entries, but in this specific bot we're only running it on things without existing isbn_13s anyway, so that's probably a worthwhile change.